BookReader/plugins/plugin.resume.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Plugin to remember the current page in a cookie
+ * @type {[type]}
+ */
+
+// Extend the constructor to add TTS properties
+BookReader.prototype.setup = (function(super_) {
```

Comment:
Suggestion: Come up with a way of adding the new setup functionality without the `super_` boilerplate.

Reply:
Note: I am going to hold off one this. 

I think some function composition would be nice here.

In some cases we want the returned value of the function to be modified in the next function.

In other cases we just want to run the two functions on the same arguments.

I am having trouble thinking of a general purpose solution for this right now, so I am going to leave these as is, and then refactor them later.

For example TTS modifies these:
```
BookReader.prototype.setup = (function(super_) {
BookReader.prototype.buildMobileDrawerElement = (function (super_) {
BookReader.prototype.buildToolbarElement = (function (super_) {
```